### PR TITLE
Checkout Form Examples

### PR DESCRIPTION
**Pull request by klaughlin**
This PR adds a page for our checkout form (inline) with examples and explanation. It also adds two new button examples: optional transaction fees and required transaction fees.

screen shot 2017-09-18 at 5 06 07 pm

screen shot 2017-09-18 at 5 06 18 pm

screen shot 2017-09-18 at 5 06 26 pm

### CODE REVIEW

File: inline.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+One of the cool things about using the inline checkout form is that it is completely customizable to fit your site. By including the link to our stylesheet before your own you can easily override css rules to add colors, change sizes, and make our form your own.
+
+Here is an example that updates the inline checkout form to match the colors of the page.
+
+{% include inline-checkout/inline-example-image.html %}
+
+The sass (compiled to css) to accomplish the above looks like this.
+
+```
```
Comment (andrewek) [inline inline.md:31]: Would it be worth also including the raw CSS that this compiles to, for folks who haven't ever used Sass or who don't have access to it for whatever reason?

File: inline.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+Each form is configurable with all the options found on our [button examples page](https://paymentspring.github.io/checkout-examples/).
+
+### Example
+
+To include a form on your website, you will need to obtain a form-id. Then copy and paste a line of javascript into your site where you would like the form to appear.
+
+```markdown
+<script src="https://checkout.paymentspring.com/js/inline.js" formId="cb7eae32763131ade35f"></script>
+```
+Where `cb7eae32763131ade35f` is your form-id.
```
Comment (andrewek) [inline inline.md]: `is` -> `is replaced with`

**Review by andrewek — APPROVED** [review]

**Comment by andrewek** [comment]
@klaughlin I think we can, at least for now, ignore the comments I have about copy and instead start updating our docs as questions roll in from users.

File: inline.md (hunk 1 above)
Comment (coreyjergensen) [reply inline.md:31]: how difficult would it be to each of these in tabs? similar to http://sass-lang.com/guide#topic-2-Sass.

**Review by coreyjergensen — APPROVED** [review]

File: inline.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+The checkout form is a simple way to seamlessly integrate the checkout button's ease of use natively into your site.
+
+The form elements are placed along side your html making the form part of your user experience and ultimately gives you the control.
+
+Each form is configurable with all the options found on our [button examples page](https://paymentspring.github.io/checkout-examples/).
+
+### Example
+
+To include a form on your website, you will need to obtain a form-id. Then copy and paste a line of javascript into your site where you would like the form to appear.
```
Comment (nminchowNelnet) [inline inline.md]: I think that we will want to extract the shared copy into  [include](https://github.com/paymentspring/checkout-examples/tree/master/_includes) files. Another option would be to set up additional [layouts](https://github.com/paymentspring/checkout-examples/tree/master/_layouts), but that may be overkill with the number of differences between these pages.

**Review by nminchowNelnet — APPROVED** [review]

**Pull request merged**